File: hdu3787/.cpp
#pragma GCC optimize(3, "Ofast", "inline")
#include <bits/stdc++.h>
using namespace std;
#define all(x) x.begin(), x.end()
int main()
{
    string line;
    while (getline(cin, line))
    {
        auto pos = line.find(' ');
        string a = line.substr(0, pos);
        string b = line.substr(pos);
        a.erase(remove(all(a), ','), a.end());
        b.erase(remove(all(b), ','), b.end());
        cout << stoll(a) + stoll(b) << endl;
    }
    return 0;
}

File: UVa455/.py
n = int(input())
for _ in range(n):
    if _:
        print()
    input()
    line = input()*2
    for i in range(1, len(line)+1):
        if all(line[j:j+i] == line[:i] for j in range(i, len(line)-i, i)):
            print(i)
            break
